base/mpfr.jl

```diff
@@ -109,8 +109,9 @@ end
 tie_breaker_is_to_even(::MPFRRoundingMode) = true
 
 const ROUNDING_MODE = Ref{MPFRRoundingMode}(MPFRRoundNearest)
+const CURRENT_ROUNDING_MODE = ScopedValue{MPFRRoundingMode}()
 const DEFAULT_PRECISION = Ref{Clong}(256)
-
+const CURRENT_PRECISION = ScopedValue{Clong}()
```

Comment:
You could set it to default precision? Or is it important that the default precision could be changed globally?

Reply:
We support it at the moment, so dropping it would be breaking, but I'm not sure how necessary it is?

Reply:
If you need to be able to change it, don't you need this to be
```
const CURRENT_PRECISION = ScopedValue{RefValue{Clong}}(Ref{Clong}())
```
since `ScopedValue` is immutable?

Reply:
I didn't mean that it's an immutable struct, I just mean that you can't actually change the value other than by introducing a new dynamic scope.

Reply:
Yes, that's correct.